x-pack/plugins/maps/common/constants.ts

```diff
@@ -62,6 +62,9 @@ export const ES_GEO_GRID = 'ES_GEO_GRID';
 export const ES_SEARCH = 'ES_SEARCH';
 export const ES_PEW_PEW = 'ES_PEW_PEW';
 export const EMS_XYZ = 'EMS_XYZ'; // identifies a custom TMS source. Name is a little unfortunate.
+export const WMS = 'WMS';
+export const KIBANA_TILEMAP = 'KIBANA_TILEMAP';
+export const REGIONMAP_FILE = 'REGIONMAP_FILE';
```

Comment:
Not 100% related to this PR, but would you mind introducing the `SOURCE_ENUM` here for all the types?

Reply:
lets wait for a separate PR for that. There is already a lot of noise in this PR.